### DISCUSSION TITLE
Getting converted pdf in stream or bytes (not output file)

### DISCUSSION THREAD

**Discussion by caroline-hughes** [gotenberg/gotenberg#810] [Q&A]
I'm able to convert to pdfs successfully, but, don't necessarily want to be creating new output files each time. Ideally, I could just receive the result as a stream or bytes type. In my case, I'm calling Gotenberg from rust, so I'd ideally be able to receive the pdf data in something like a Vec<u8> response. 

Has anyone asked about this before? I saw people asking about sending bytes, but not receiving

**Comment by gulien** [comment]
Hello @caroline-hughes,

I’m a bit rusted on this subject (no pun intended), but can’t you just extract the content of the body instead of creating an output file?

**Reply by caroline-hughes** [reply]
Yes - I was confused! Ended up figuring it out. Thanks
